src/App.vue

```diff
@@ -901,4 +902,8 @@ export default class App extends Mixins(
   background-color: rgb(33, 33, 33); // grey darken-4
   border-color: rgb(33, 33, 33); // grey darken-4
 }
+
+.color-white {
+  color: white;
+}
```

Comment:
This now matches the style/accessibility of the alert banner in Auth Web, ie:

![image](https://github.com/user-attachments/assets/c95f98d8-6e73-4684-8eac-32a764f73f17)